apps/meteor/tests/end-to-end/api/livechat/methods/changeLivechatStatus.ts

```diff
@@ -0,0 +1,226 @@
+/* eslint-env mocha */
+
+import type { ILivechatAgent, IUser } from '@rocket.chat/core-typings';
+import type { Response } from 'supertest';
+import { expect } from 'chai';
+
+import { getCredentials, request, credentials, methodCall } from '../../../../data/api-data';
+import { createAgent } from '../../../../data/livechat/rooms';
+import { updatePermission, updateSetting } from '../../../../data/permissions.helper';
+import { password } from '../../../../data/user';
+import { createUser, getMe, login } from '../../../../data/users.helper';
+import { disableDefaultBusinessHour, makeDefaultBusinessHourActiveAndClosed } from '../../../../data/livechat/businessHours';
+
+describe('Omni-Method - changeLivechatStatus', function () {
```

Comment:
```suggestion
describe('livechat:changeLivechatStatus', function () {
```

So, if you look for the method by name, you find the tests associated 👀